tennis-scheduler-web-app/src/components/TennisCourtInfo.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import AddTimeslot from "./AddTimeslot";
+import axios from "axios";
+import "../styles/courts.css";
+
+export const TennisCourtInfo = ({ id, image, name, surfaceType, description }) => {
+
+  const [timeslotErrors, setTimeslotErrors] = useState("");
+  const [buttonName, setButtonName] = useState("Add timeslot");
+  const [showAddTimeslot, setAddTimeslot] = useState(false);
+
+  const addTimeslot = async (timeslot) => {
+
+    let Timeslot = {
+      dateStart: `${timeslot.timeslotDate}T${timeslot.startTime}:02.174Z`,
+      dateEnd: `${timeslot.timeslotDate}T${timeslot.endTime}:02.174Z`,
+      personId: 1,
```

Comment:
Why person id is hardcoded?

Reply:
Because only logged in user can reserve timeslot. 
In this case I still don't have connected project with security.